controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -296,7 +311,7 @@ func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) (re
 	// we do not check if it is a provider before checking storageConsumers CR because of corner case
 	// where user can mark instance.Spec.AllowRemoteStorageConsumers as false and mark CR for deletion immediately.
 	// Which will trigger a deletion without having instance.Spec.AllowRemoteStorageConsumers as true.
-	err := r.verifyNoStorageConsumerExist(sc)
+	err := r.verifyNoStorageConsumerExist(sc, true /* skip local */)
```

Comment:
As stated in prev comment, we don't need this special case.

Reply:
done.